tests/tasks/git-skip-empty-commit-push.yaml

```diff
@@ -0,0 +1,32 @@
+- name: "{{ testname }} - make sure we are on the branch {{ branch }}"
+  shell: git checkout -b {{ branch }} || git checkout {{ branch }}
+  args:
+    chdir: /{{git_repo_name}}
+
+- name: "{{ testname }} - create empty commit in {{git_repo_name}}"
+  command: git commit -m "empty commit but [skip deploy]" --allow-empty
```

Comment:
I think it would be a more real test if we run a regular commit/deploy first, then test a 2nd commit which does [skip deploy]